internal/build/src/tasks/helper.ts

```diff
@@ -27,18 +27,25 @@ const typeMap = {
   vue: ['Component', 'VNode', 'CSSProperties', 'StyleValue'],
 }
 
-const reComponentName: ReComponentName = (title) =>
-  `el-${hyphenate(title).replace(/[ ]+/g, '-')}`
+const removeTag = (str: string) => {
+  return str.replaceAll(/\^\([^)]*\)/g, '').trim()
```

Comment:
Are `replaceAll` and `g` duplicates?

Reply:
That's right, you can refer to [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#:~:text=If%20pattern%20is%20a%20regex).